js/packages/web/src/views/artCreate/index.tsx

```diff
@@ -638,11 +638,12 @@ const InfoStep = (props: {
           {props.attributes.image && (
             <ArtCard
               image={image}
-              animationURL={animation_url}
+              animationURL={props.attributes.animation_url}
               category={props.attributes.properties?.category}
               name={props.attributes.name}
               symbol={props.attributes.symbol}
               small={true}
+              artView={props.files.length>1?false:true}
```

Comment:
```suggestion
              artView={!props.files.length}
```

Reply:
I think it should be 
`artView={!props.files.length>1}`